pages/your_domain/structural_bioinformatics.md

```diff
@@ -48,3 +48,19 @@ Researchers in the field should be able to find predictions of macromolecular st
   * input data needed for the modelling steps. For instance in homology modelling this could include the {% tool "pdb" %} identifiers for the template structures used for modelling and their alignments to the target protein;
   * model quality estimates.
 * If necessary, accompanying data can be provided in separate files using different file formats. The files can be added to ModelArchive depositions and referred to in the PDBx/mmCIF ModelArchive extension format.
+
+## Storing and sharing experimentally solved structures
+ 
+### Description
+
+Experimentally solved atomic structures of molecules can be obtained by several methods, such as X-ray crystallography,  Nuclear Magnetic Resonance (NMR) spectroscopy, and 3D Electron Microscopy. Here you can find useful tools and guides for storing and sharing structure models based on these methods.
+* Structure models resulting from experimental methods are broadly in the {% tool "pdb" %} under the [CC0 1.0 Universal (CC0 1.0) Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/).
+* Additionally, raw and intermediate data associated with the structure model can be also published in different curated data archives depending on the methods used. Raw EM data and processed 3D volumes and tomograms can be stored in {% tool "empiar" %} and {% tool "emdb" %}, respectively. Raw data from NMR studies can be stored in {% tool "bmrb" %}. For X-ray diffraction experiments raw data can be stored in {% tool "irrmc" %} and {% tool "sbgrid-data-bank" %}. Data submitted in these repositories can be cross-referenced to related PDB entries.
+* To extend molecular context, structure models can be visualized and analyzed along with respective volume maps and {% tool "uniprot" %} sequences using data aggregators such as {% tool "3dbionotes" %}, either if they have been published in PDB and EMDB or if not. A bunch of biochemical, biomedical and validation annotations would be mapped on the structure model coordinates and sequence to let the user better understand macromolecular function and interactions. Users can also use the [COVID-19 Structural Hub](https://3dbionotes.cnb.csic.es/ws/covid19), a dedicated summary view for all published SARS-CoV-2 structure models. 
```

Comment:
As we are adhering to the UK spelling, verbs ending in "ized" (e.g. visualized and analyzed) should end in "ised" instead. This is mentioned here: https://rdmkit.elixir-europe.org/style_guide

Reply:
Done!